tests/test_mmar_download.py

```diff
@@ -94,16 +94,16 @@
             "item": "clara_pt_pathology_metastasis_detection",
             "map_location": "cuda" if torch.cuda.is_available() else "cpu",
         },
-        "TorchVisionFullyConvModel",
+        "TorchVisionFCModel",
```

Comment:
May I know why change to this deprecated API?

Thanks.

Reply:
Hi Nic, what is 'change to deprecated api'?

Reply:
Ah, I misunderstood the source code, `TorchVisionFullyConvModel` is the deprecated API.

Thanks.